build/webpack.client.config.js

```diff
@@ -19,26 +19,31 @@ const config = merge(base, {
       'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV || 'development'),
```

Comment:
the default `NODE_ENV` is not same as `mode`? And it can be removed.

Reply:
thanks, you right